packages/flutter_test/lib/src/binding.dart

```diff
@@ -1433,8 +1453,11 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     switch (source) {
       case TestBindingEventSource.test:
         if (!renderView._pointers.containsKey(event.pointer)) {
-          assert(event.down);
-          renderView._pointers[event.pointer] = _LiveTestPointerRecord(event.pointer, event.position);
+          assert(event.down || event is PointerAddedEvent);
+          if (event.down) {
+            renderView._pointers[event.pointer] = _LiveTestPointerRecord(
+              event.pointer, event.position);
```

Comment:
please see style guide regarding line wrapping (if there's a newline after `(` there should be one before `)`)